tutorials/appengine-pusher/index.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Using Pusher on Google App Engine Standard Java 7 Environment
+description: Learn how to use Pusher in place of the deprecated Channels API on App Engine Standard.
+author: jabubake
+tags: App Engine, Pusher, Channels API, Java
+date_published: 2017-08-09
+---
+
+This tutorial demonstrate how to use [Pusher](https://pusher.com) on [Google App Engine](/appengine).
+Pusher is a hosted API for sending real-time, bi-directional messages via WebSockets to apps and
+other Internet-connected devices.
+
+Pusher's real-time functionality replicates the deprecated [Channel API](/appengine/docs/java/channel/)
+and is useful for applications that send information in real time,
+such as collaborative applications, multi-player games, and chat rooms.
+
+Using Pusher is a better choice than polling in situations where updates can't be predicted or 
+scripted, such as when relaying information between human users, or when events aren't generated
+systematically.
+
+Using WebSockets, Pusher is able to deliver server-side events to clients.
+Pusher also offers  HTTP REST API based server-side SDKs to enable sending events from server to a
+public or secured channel.
+
+In this tutorial you’ll learn how to complete the following tasks on the server:
++  Set up your server to use the Pusher service
++  Authenticate subscriptions to secure channels
++  Send messages over the channel
+
+You'll also learn how to complete the following tasks in the web browser:
++  Set up your client to use the Pusher service
++  Subscribe to Pusher events
++  Subscribe to channels
++  Send messages to the server so they can be passed on to remote
+   clients. 
+    
+## Before you begin  
+
+1.  Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/) and run:
+    ```
+       gcloud init
+    ```
+    If this is your first time creating an App Engine application, run the following command
```

Comment:
I suggest make this its own step. We have instructed the reader to create a new project, so it is the "first time" they will have created an App Engine application. If they meet these prerequisites, they will likely choose to skip this section.

If you agree, change sentence to:

"1. Create an App Engine application:"